### DISCUSSION TITLE
Student Developer Pack Problem

### DISCUSSION THREAD

**Discussion by LIN-WEI-CHIEH0304** [community/community#57804] [GitHub Education]
### Hi everyone,

I tried more than 20 times, but I can't pass it,  I got the reply like this : 
You are unlikely to be verified until you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name as it appears in your academic affiliation document plus a short bio.
and my profile : https://github.com/riku0304
I don't know where I can correct..

**Comment by Hamzalopode** [comment]
You need to add the information in your account settings not the profile I think. Also don't forget to add your school email in your github account. ( settings -> email)

**Reply by LIN-WEI-CHIEH0304** [reply]
I did it but still not work

**Reply by Hamzalopode** [reply]
Then you probably did something wrong in the first steps, try redoing step by step this: https://docs.github.com/fr/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/apply-to-github-global-campus-as-a-student

**Reply by LIN-WEI-CHIEH0304** [reply]
Okay, thank for replying

**Reply by Hamzalopode** [reply]
Np, good luck

**Reply by dhrumibhatia** [reply]
what if the school id is not given to the student by school so can i provide my email id that i have given to the school?

**Comment by faketi101** [comment]
you should submit your educational verification like college id card or like stuffs
also you have to complete your github profile
